packages/eas-cli/src/metadata/apple/tasks/age-rating.ts

```diff
@@ -0,0 +1,42 @@
+import { AgeRatingDeclaration } from '@expo/apple-utils';
+import assert from 'assert';
+import chalk from 'chalk';
+
+import Log from '../../../log';
+import { logAsync } from '../../utils/log';
+import { AppleTask, TaskDownloadOptions, TaskPrepareOptions, TaskUploadOptions } from '../task';
+
+export type AgeRatingData = {
+  /** The app age rating declaration for the app version */
+  ageRating: AgeRatingDeclaration;
+};
+
+export class AgeRatingTask extends AppleTask {
+  name = (): string => 'age rating declarations';
+
+  async prepareAsync({ context }: TaskPrepareOptions): Promise<void> {
+    assert(context.version, `App version information is not prepared, can't update age rating`);
```

Comment:
the same as in other PR, if this is error handling it shouldn't use asserts(the same applies to other asserts)

Reply:
Alright, so I think these asserts are right:

- If `Task2` is being executed before `Task1`, and `Task2` requires the data from `Task1` -> that would be a programmer error
- If `Task1` does not set the data required for `Task2` -> that would be a wrong permissive `null` value that would always fail `Task2` (either programmer error, or wrongfully handled exception from ASC)

I'll let this be like this for now